### ISSUE TITLE
Improve theme documentation

### ISSUE THREAD

**Issue by jrcii** [helix-editor/helix#5090]
I'm giving Helix a run and immediately ran into a gotcha with trying to set a theme. My experience may be related to the MacPorts build:

```
< install helix
> done
< helix
> Command not found
< port contents helix
> ... hx
"Oh the binary isn't helix (nor is there an alias/link for that command) it's hx, ok"
"Oh I see the port includes a bunch of themes, cool, how do I turn it on"
*Googles docs* "Do :theme <name>" 
< hx
< :theme onedark
> Theme not found
"Hm that's weird they were all in the list"
* Research *
< ln -s /opt/local/share/helix/runtime/themes ~/.config/helix/themes
```
Now themes work.

What would have saved me some time is if the docs explicitly explained that the install (at least in the case of MacPorts, I'm not sure what other package managers) may include some themes which are not available by default and must be copied to the theme directory first, or optionally the sym link strategy above to enable all of them.

**Comment by pascalkuthe** [comment]
I don't think this is a documentation issue or a helix issue at all.
The runtime directory is critical for the function of helix, not just for helix and a core part of the installation.
Therefore package managers usually wrap helix in a wrapper script that sets `HELIX_RUNTIME`.
It appears that macports does not do this. I took a look at their package file and wanted to make a PR.
However their PR template requires a bunch of verification steps that require a mac (which I don't have).

Instead I submitted a ticket https://trac.macports.org/ticket/66428#ticket.
I think this should be fixed in mac ports,  I don't think it makes sense for us to document the particularities of every package manager out there.
